desktop/SmesherManager.ts

```diff
@@ -38,18 +42,35 @@ class SmesherManager extends AbstractManager {
 
   private genesisID: string;
 
+  private netName: string;
+
   constructor(
     mainWindow: BrowserWindow,
     configFilePath: string,
-    genesisID: string
+    genesisID: string,
+    netName: string | undefined
   ) {
     super(mainWindow);
     this.smesherService = new SmesherService();
     this.smesherService.createService();
     this.configFilePath = configFilePath;
     this.genesisID = genesisID;
+    this.netName = netName || '';
   }
 
+  updateConfig = async (netName: string, config: NodeConfig) => {
+    const customNodeConfig = await loadOrCreateCustomConfig(netName);
+    const nodeConfig = mergeDeepLeft(customNodeConfig, config);
+
+    if (R.equals(nodeConfig, await this.loadConfig())) {
```

Comment:
Do we really need to load it once again?
```suggestion
    if (R.equals(nodeConfig, config)) {
```